internal/rules/resources.go

```diff
@@ -104,8 +104,9 @@ func (r *resources) Grant(resource string, teamID uint64, operations []string, v
 	row := types.Rules{
 		TeamID:   teamID,
 		Resource: resource,
-		Value:    string(value),
+		Value:    int(value),
```

Comment:
@mitjaziv can you please move Access to the types package here, so we can avoid the type conversion fully